### DISCUSSION TITLE
Vertical scrollbar in Status History Panel

### DISCUSSION THREAD

**Discussion by cicciopizza** [grafana/grafana#48697] [Dashboards, Panels & Visualizations]
It would be useful to have a vertical scrollbar in status history panel, when the data series are many and the display is not clear:
![Selezione_029](https://user-images.githubusercontent.com/26459829/166680375-6ce2ecfd-efdc-436b-99a1-ae982e663576.png)

**Comment by mohammadne** [comment]
is there any update?

**Reply by leeoniya** [reply]
no update yet but we'll be looking at this in the next few months. there is a similar highly upvoted request for StateTimeline panel, which shares the same rendering core as StatusHistory.

out of curiosity, what's the use scenario for rendering rows that are hidden without extra user interaction? i can understand this in Explore mode, when you're actively investigating and digging through data in search of something, but cannot understand the usage patten for Dashboards, is it mostly to avoid it looking broken visually? 🤔 

implementation-wise it would be much easier for us to add button pagination than to add scrolling, which requires figuring out how to make the time axis sticky.

**Comment by natellium** [comment]
Closing in favor of GH issues: https://github.com/grafana/grafana/issues/80138
